packages/create-docusaurus/templates/facebook/docusaurus.config.js

```diff
@@ -148,8 +148,11 @@ const config = {
           alt: 'Meta Open Source Logo',
           // This default includes a positive & negative version, allowing for
           // appropriate use depending on your site's style.
-          src: 'img/meta_opensource_logo_negative.png',
+          src: '/img/meta_opensource_logo_negative.svg',
```

Comment:
Do we need this leading slash?

Reply:
no :D 

With/without it's the same afaik, and historically many config elements don't have one

But I'd prefer to always have one in the future to make it more explicit it's not a relative path. Does it make sense?

Reply:
Absolute paths, IMO, are always relative to the domain. The fact that we automatically prepend the base URL is quite magical. But I'm fine with either since it's not strictly a relative path either.

Reply:
yes 🤪 I know, no ideal choice there. I'd prefer to have leading `/` everywhere, but this is just my personal opinion, and Docusaurus v1 did not have any leading `/` in docs/configs either so 🤷‍♂️ . I can revert it if needed.

@yangshun you remember why historically we don't have any `/`?